app/api/admin_translations.py

```diff
@@ -10,7 +10,6 @@
 translations_dir = 'app/translations'
 
 @admin_blueprint.route('/', methods=['GET'])
-@is_admin
```

Comment:
Why did you remove the following decorator ?

Reply:
Ah! I was testing the translation thing on my local about which @mrsaicharan1 just opened the issue. Forgot to fix it. Fixing it now